src/main/java/resourceloader/EventLoader.java

```diff
@@ -89,6 +89,7 @@ private void createNewDataFile() {
             Files.createDirectories(fileDirectory);
             Files.createFile(Paths.get(filePath));
             Files.createFile(Paths.get(Ui.FILE_PATH_STUDYAREAS));
+            Files.createFile(Paths.get(Ui.FILE_PATH_DICTIONARY));
```

Comment:
This fix is okay for now, Try to move these 2 lines to the StudyAreaLoader Class later.